rpush.gemspec

```diff
@@ -36,23 +36,24 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'net-http-persistent'
   s.add_runtime_dependency 'net-http2', '~> 0.18', '>= 0.18.3'
   s.add_runtime_dependency 'jwt', '>= 1.5.6'
-  s.add_runtime_dependency 'activesupport', '>= 5.2'
+  s.add_runtime_dependency 'activesupport', '>= 5.2', '< 7.1.0'
```

Comment:
Hi, is there a reason you have to pin this to `< 7.1.0`?

Reply:
https://github.com/rpush/rpush/pull/675